hw_solution.rb

```diff
@@ -0,0 +1,151 @@
+
+students = [
+    {
+        first_name: 'John',
+        last_name: 'Doe'
+    },
+    {
+        first_name: 'Jane',
+        last_name: 'Doe',
+    },
+    {
+        first_name: 'Jennifer',
+        last_name: 'Aniston',
+    }
+  ]
+  
+  upper_case_full_names = []
+
+
+students.each do |a|   # this is an array of arrays of hashes
+        # this is an array of hashes
+    #    puts a.fisrt_name
+  #      upper_case_full_names.push(a[:first_name].upcase + a[last_name].upcase)
```

Comment:
Great work; but I think you meant here ```a[:last_name]``` not ```a[last_name]```, also the space between ```a[:first_name]``` and ```a[:last_name]``` so the code will be: 
```rb
students.each do |a|  
  upper_case_full_names.push(a[:first_name].upcase + " " + a[:last_name].upcase)
end
```